tsdb/encoding/encoding.go

```diff
@@ -260,6 +268,19 @@ func (d *Decbuf) Byte() byte {
 	return x
 }
 
+func (d *Decbuf) ConsumePadding() {
+	if d.E != nil {
+		return
+	}
+	for len(d.B) > 1 && d.B[0] == '\x00' {
+		d.B = d.B[1:]
+	}
+	if len(d.B) < 1 {
+		d.E = ErrInvalidSize
+		return
```

Comment:
Not needed

Reply:
Removed, was unsure whether to go for clarity or succinctness here.